### PR TITLE
[accordion] feat: reduce accordion bundle size

### PR DESCRIPTION
**Pull request by maimonav**

### CODE REVIEW

File: packages/plugin-accordion/web/src/defaults.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
-import { ArrowIcon } from './icons';
-import { EditorState, __convertToRawWithoutVersion } from 'wix-rich-content-editor';
+import {
```
Comment (shaulgo) [inline packages/plugin-accordion/web/src/defaults.ts:1]: revert

File: packages/plugin-accordion/web/src/defaults.ts [hunk 2]
```diff
@@ -1,5 +1,7 @@
...
+  EditorState,
+  __convertToRawWithoutVersion,
```
Comment (shaulgo) [inline packages/plugin-accordion/web/src/defaults.ts:3]: don't load defaults in viewer

File: packages/plugin-accordion/web/src/defaults.ts (hunk 1 above)
Comment (maimonav) [reply packages/plugin-accordion/web/src/defaults.ts:1]: done.

File: packages/plugin-accordion/web/src/defaults.ts (hunk 2 above)
Comment (maimonav) [reply packages/plugin-accordion/web/src/defaults.ts:3]: done.

Comment (maimonav) [reply packages/plugin-accordion/web/src/defaults.ts:3]: It appears that default is indeed needed in viewer, consider this:
https://github.com/wix-incubator/rich-content/blob/feat%2FreduceAccordionBundleSize/packages/plugin-accordion/web/src/viewer.ts#L14

So I think it's best to give default value without using `EditorState` & `__convertToRawWithoutVersion` at all.

**Pull request merged**